### PR TITLE
Adds IPData geolocation

### PR DESCRIPTION
**Pull request by duartemateus**

### CODE REVIEW

File: config/config.json.default [hunk 1]
```diff
@@ -528,6 +528,10 @@
       "uri": "https://zetalytics.com/",
       "editable": "true",
       "comment": ""
+    },
+    "geolocation_api_key": {
```
Comment (shpendk) [inline config/config.json.default:532]: I'd rename this to `ipdata_api_key` just so we know the provider

**Review by shpendk — CHANGES_REQUESTED** [review]
Looks good to me for the one inline issue, and regarding the file `data/web_accounts_list/GeoLite2-City.mmdb` I was wondering if this is still needed? If not, we should remove it

File: config/config.json.default (hunk 1 above)
Comment (duartemateus) [reply config/config.json.default:532]: will do, wasn't too sure if we wanted to "obfuscate" the provider we were using!

**Comment by duartemateus** [comment]
> Looks good to me for the one inline issue, and regarding the file `data/web_accounts_list/GeoLite2-City.mmdb` I was wondering if this is still needed? If not, we should remove it

Think we can bin that I'll ping you just to do some testing after removal as I have encountered some issues on my side when attempting to get rid of all usages of maxmind, including the Gem and so on.

**Review by shpendk — APPROVED** [review]
looking good.

**Pull request merged**